### DISCUSSION TITLE
向左拖拽的时候，能否保持画布不向左缩呢

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [antvis/S2#2128] [Q&A]
![image](https://user-images.githubusercontent.com/15354455/226781421-c7e56209-f7bd-4faf-b9a5-7b91905d3cd3.png)

就是拖拽x点的时候，A的区域变大，x->y 的距离变大，而不是整体向左收缩呢。

**Comment by lijinke666** [comment]
单元格的热区分左右, x 点对应的拖拽热区是 列头[type] 的热区, 所以你往左拖拽, 其实是调整 type 的宽度, 你要变大 A 区域, 应该是拖 y 点, 目前设计是如此, 是不支持的, 你想实现左热区调整右相邻的单元格的宽度, 需要参考拖拽相关代码自定义交互

![image](https://user-images.githubusercontent.com/21015895/229011906-a7652e75-c586-44f4-a099-99b34ea6b813.png)
